containers/Makefile

```diff
@@ -59,6 +59,8 @@ concent-api-clean:
 build/concent-api/concent-api.tar: $(CONCENT_DIR)
 	mkdir --parents $(dir $@)
 	package-builder/build-concent-api.sh "$(CONCENT_DIR)" "build/concent-api"
+	tar --extract --file build/concent-api/concent-api.tar \
+	--directory build/concent-api/ concent_api/requirements.lock --strip=1
```

Comment:
Creating the `requirements.lock` file must be a separate target. Otherwise if the command fails make will not notice it.